### PR TITLE
Create open-service-mesh-at-kubecon-na-22.md

### PR DESCRIPTION
**Pull request by karenhchu**
Signed-off-by: Karen Chu <karen.chu@microsoft.com>

### CODE REVIEW

**Comment by karenhchu** [comment]
@phillipgibson @bridgetkromhout - can y'all take a look? thank you!

**Comment by phillipgibson** [comment]
Looks good.

File: content/blog/open-service-mesh-at-kubecon-na-22.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## Open Service Mesh Booth in Project Pavilion 
+
+WED OCT 26: 10:30 – 15:30 ET
+
+THUR OCT 27: 10:30 – 14:30 ET
+
+FRI OCT 28: 10:30 – 13:00 ET
+
+LOCATION: on the backside of the Open Policy Agent (OPA) booth, near the Cloud Native Buildpacks/Tinkerbell/Istio booths
```
Comment (bridgetkromhout) [inline content/blog/open-service-mesh-at-kubecon-na-22.md:43]: > backside

opposite side

File: content/blog/open-service-mesh-at-kubecon-na-22.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+OSM maintainers [Phill Gibson](https://twitter.com/phillipgibson), [Keith Mattix](https://twitter.com/keithmattix), and [Jackie Elliot](https://twitter.com/jcelliott17) will be hanging out at the OMS project booth the first half of each day. The project would love to learn from the community how OSM is being used, what pain points exists, and what users are looking for in the future. Help us help you by sharing your thoughts in our [OSM User Survey](https://forms.office.com/r/WCDvxK6WJ1), even if you're not able to drop by the booth.
```
Comment (bridgetkromhout) [inline content/blog/open-service-mesh-at-kubecon-na-22.md:45]: > OMS project booth

**OSM** project booth

File: content/blog/open-service-mesh-at-kubecon-na-22.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+THUR OCT 27: 10:30 – 14:30 ET
+
+FRI OCT 28: 10:30 – 13:00 ET
+
+LOCATION: on the backside of the Open Policy Agent (OPA) booth, near the Cloud Native Buildpacks/Tinkerbell/Istio booths
+
+OSM maintainers [Phill Gibson](https://twitter.com/phillipgibson), [Keith Mattix](https://twitter.com/keithmattix), and [Jackie Elliot](https://twitter.com/jcelliott17) will be hanging out at the OMS project booth the first half of each day. The project would love to learn from the community how OSM is being used, what pain points exists, and what users are looking for in the future. Help us help you by sharing your thoughts in our [OSM User Survey](https://forms.office.com/r/WCDvxK6WJ1), even if you're not able to drop by the booth. 
+
+Now for the fun part – the OSM booth will be giving away new Open Service Mesh stickers and custom OSM + Carhartt wool socks to warm while in Detroit!
```
Comment (bridgetkromhout) [inline content/blog/open-service-mesh-at-kubecon-na-22.md:47]: > Carhartt wool socks to warm while in Detroit

Carhartt wool socks to **stay** warm while in Detroit

**Review by bridgetkromhout — APPROVED** [review]

**Pull request merged**